src/System.Text.Encoding/tests/Decoder/DecoderConvert2.cs

```diff
@@ -155,6 +155,7 @@ public void PosTest5()
 
         // PosTest6: Call Convert to convert a ASCII character array with Unicode encoder
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "coreclr #27191 is not fixed in netfx.")]
```

Comment:
We don't appear to run these tests on netfx:
https://github.com/dotnet/corefx/blob/90b15f766dc78bcb9246eff6b64c02220dacb08f/src/System.Text.Encoding/tests/Configurations.props#L4-L5
so I wouldn't think this attribution would be necessary.